### ISSUE TITLE
iOS weird UI offset (scrollView behaviour), haven't tested on Android.

### ISSUE THREAD

**Issue by sooxiaotong** [underscopeio/react-native-facebook-account-kit#144]
![ezgif-4-76b1d7abc1](https://user-images.githubusercontent.com/13378059/42725690-b840894c-87ba-11e8-9082-fdbe8e163bf8.gif)

> Device: iPhone X

```
"react-native-facebook-account-kit": "^0.10.1",
"react-native": "0.55.4"
```

**Comment by billel-boudchicha** [comment]
Same here

**Comment by Im-Goutham** [comment]
Same issue occuring for me.Got this issue in iPhone X simulator.

**Comment by manurana** [comment]
This should go away on the device .. This has been my experience.

**Comment by gaguirre** [comment]
Thanks @manurana for the info.
@sooxiaotong @billel-boudchicha @Im-Goutham could you confirm this is happening on simulator only?

**Comment by sooxiaotong** [comment]
@gaguirre I don't have an iPhone, I had tested this in simulator only.

**Comment by oleksiiLevytskyi** [comment]
In my case it looks the same, but on android devices input shows without country name only code like this
<img width="315" alt="2018-09-26 13 45 04" src="https://user-images.githubusercontent.com/34065268/46075221-73fac380-c192-11e8-82e8-8d6a0c87f8d1.png">
how can I change it?

**Comment by azamouchi** [comment]
i'm having the same issue here, using release v0.10.1 of the module, and yes i confirm that it's only happening on simulators, but this is critical, because when you submit the app to app store for review, they test your app on a simulator, and when they faced this issue they reject my build because of a 
 **UI design and user experience issue**  😞

I tried to explain that this happens only on a simulator and they should test the app on a real device, but they didn't accept my argument.

Any fixe planned for this issue 🙏 🙏  ? 

![ezgif-1-2a4b254350e2](https://user-images.githubusercontent.com/6905567/49868343-7245b100-fe0d-11e8-91c2-1a306ff0b3b4.gif)

**Comment by manurana** [comment]
I think I have a diagnosis and a fix for this. This seems to be an issue with the FB iOS Account Kit SDK which was fixed at some point of time. The problem is that if you added react-native-facebook-account-kit to your RN app using the drag-and-drop method, the new FB iOS SDK never gets updated even if you update the version of react-native-facebook-account-kit.

Here is what I did to resolve it. First I had to reverse the drag-and-drop linking that I had initially done.
1. Removed `AccountKit.framework` and `AccountKitStrings.bundle` from my xCode project. Just remove the reference, which is the default option.
2. Ran `react-native unlink react-native-facebook-account-kit`

Now my project is (hopefully) in the state referred to in the [Semi-automatic install method](https://github.com/underscopeio/react-native-facebook-account-kit#semi-automatic-suggested) (which asks you to run`react-native link`, and undo any changes in the ios folder)

Now I just proceeded with the POD install method, which is step 2 in the [Semi-automatic install method](https://github.com/underscopeio/react-native-facebook-account-kit#semi-automatic-suggested) under `iOS (via Cocoapods)`

3. Add the following to your podfile : `pod 'RNAccountKit', :path => '../node_modules/react-native-facebook-account-kit/ios'`
4. run `pod install` in the ios directory

Now you should be able to see the new UI, and this problem should be resolved

**Comment by azamouchi** [comment]
@manurana ,Yes, i've had to do the same thing tpp (without using cocoapods), the issue is inside **AKFViewController** itself, and we have no control of it, here is how i proceeded to fix the issue : 
- Downloaded the latest version of FBSDK for iOS (4.39.0 for now)
- Replaced the **AccountKit.framework** and **AccountKitStrings.bundle** in both react-native-facebook-account-kit module folder and my Xcode project (i've had to use my own fork of react-native-facebook-account-kit for this)
- Now when i run my app i can see the new AccountKit UI which does not have this issue.

**Comment by jpgarcia** [comment]
Closing this. Should be fixed on `v1.1.0` as described here https://github.com/underscopeio/react-native-facebook-account-kit/pull/164
